src/store/src/Bridge/Redis/Store.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Store\Bridge\Redis;
+
+use Symfony\AI\Platform\Vector\Vector;
+use Symfony\AI\Platform\Vector\VectorInterface;
+use Symfony\AI\Store\Document\Metadata;
+use Symfony\AI\Store\Document\VectorDocument;
+use Symfony\AI\Store\Exception\RuntimeException;
+use Symfony\AI\Store\ManagedStoreInterface;
+use Symfony\AI\Store\StoreInterface;
+use Symfony\Component\Uid\Uuid;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+class Store implements ManagedStoreInterface, StoreInterface
+{
+    public function __construct(
+        private readonly \Redis $redis,
+        private readonly string $indexName,
+        private readonly string $keyPrefix = 'vector:',
+        private readonly Distance $distance = Distance::Cosine,
+    ) {
+    }
+
+    /**
+     * @param array{vector_size?: positive-int, index_method?: string, extra_schema?: list<string>} $options
+     *
+     * - For Mistral: ['vector_size' => 1024]
+     * - For OpenAI: ['vector_size' => 1536]
+     * - For Gemini: ['vector_size' => 3072] (default)
```

Comment:
```suggestion
```

Reply:
Why? It's super useful

Reply:
I think it's not possible to maintain this in the long run, but fine to keep it for now